Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.4
 
-RUN pip install -U pipenv==8.2.7
+RUN pip install -U pipenv==8.2.7 \
+  && pip install -U awscli==1.11.174
```

Comment:
Why are the versions specified in-line here rather than in a requirements.txt file?

Reply:
The aren't dependencies of the application. they are dependencies of the docker environment

Reply:
Because these are dependencies required to bootstrap survey runner - they aren't required to run survey runner.

Reply:
gotchya